src/js/admin/wc-square-admin-products.js

```diff
@@ -31,6 +31,39 @@ jQuery( document ).ready( ( $ ) => {
 
 	// products quick edit screen.
 	if ( 'edit-product' === pagenow ) {
+		// Add notice near stock field in bulk edit when Square SOR and inventory sync is enabled.
+		if ( wc_square_admin_products.is_square_sor && wc_square_admin_products.is_inventory_sync_enabled ) {
```

Comment:
```suggestion
		if ( wc_square_admin_products.is_inventory_sync_enabled ) {
```
As discussed earlier, Square is source of truth for inventory updates regardless of the SOR.